doc/introduction/interfaces.rst

```diff
@@ -27,6 +109,13 @@ In PennyLane, an interface is declared when creating a :class:`~.QNode`, e.g.,
     If no interface is specified, PennyLane will default to the NumPy interface (powered by the
     `autograd <https://github.com/HIPS/autograd>`_ library).
 
+.. note::
+
+    QNodes with an interface will always incur a small overhead on evaluation. If you do not
+    need to compute quantum gradients of a QNode, specifying ``interface=None`` will remove
+    this overhead and result in a slightly faster evaluation. However, gradients will no
+    longer be available.
+
```

Comment:
I'm on the fence over whether to include this or not. It might be overcomplicated for the quickstart landing page.

Reply:
It's a note, it will be great. I use quickstart all the time as a reference.

Reply:
I'd move it to the bottom of the page though?

Reply:
I like it too!

Reply:
Have moved it to the bottom of the page!